### PR TITLE
Remove log statements about file uploads

### PR DESCRIPTION
**Pull request by premun**
File uploads don't happen until we send the job. Before that, we only add them to the `IJobDefinition`.

I noticed this while uploading from my PC where it immediately says Uploading, Finished, but then takes 5 minutes saying Sending where that is when we really upload.

There are other statements about adding correlation payload in between with lower priority so removing these makes sense.

image

### CODE REVIEW

**Comment by lpatalas** [comment]
You're right that these messages are lying about uploading files but I'm not sure if we shouldn't leave at least the first of them but reword it to `Adding payloads for...` and maybe set lower importance? I can imagine a situation in which no correlation payloads were added and then you won't be able to grep logs for `Adding Correlation Payload` but you would still see `Adding payloads for...` in the logs.

Maybe I'm overthinking this but I prefer to have a bit too much logging rather than too little. ;)

**Review by lpatalas — APPROVED** [review]

**Review by alexperovich — APPROVED** [review]
